jakarta-xml-binding/bin/run-tck.sh

```diff
@@ -108,15 +153,37 @@ BASE_MODULE_DIR="${JBOSS_HOME}/modules/system/layers/base"
 # JAXB API
 TCK_CLASS_PATH=""
 addToClassPath "${BASE_MODULE_DIR}/jakarta/xml/bind/api/main/"
-# Legacy implementation module
-addToClassPath "${BASE_MODULE_DIR}/com/sun/xml/bind/main/"
-# New implementation module
 addToClassPath "${BASE_MODULE_DIR}/org/glassfish/jaxb/main/"
-addToClassPath "${BASE_MODULE_DIR}/org/apache/xerces/main/"
+#addToClassPath "${BASE_MODULE_DIR}/org/glassfish/jaxb/main/" "jaxb-xjc*.jar"
+#addToClassPath "${BASE_MODULE_DIR}/org/glassfish/jaxb/main/" "istack-commons-runtime-*.jar"
+#addToClassPath "${BASE_MODULE_DIR}/org/glassfish/jaxb/main/" "txw2*.jar"
 addToClassPath "${BASE_MODULE_DIR}/jakarta/activation/api/main"
+addToClassPath "${BASE_MODULE_DIR}/org/eclipse/angus/activation/main"
 # Required for the com.sun.tools.jxc.SchemaGenerator. The options look for -cp or -classpath passed to the entry point.
 # However, the TCK runner does not pass that argument so it falls back to the CLASSPATH environment variable.
 CLASSPATH="${TCK_CLASS_PATH}"
```

Comment:
That's ok for now but it's tempting to bring it back some time, since that seems to be how other impls test. So if we ever have another exercise like the last couple days we won't have the whole tangent of how others test to navigate.

I have a branch that I could use resurrect this at some point. I dealt with the TCK_CLASS_PATH thing on it.